README.md

```diff
@@ -1,5 +1,48 @@
 # Open vSwitch CNI plugin
 
+This plugin allows user to define Kubernetes networks on top of Open vSwitch bridges available on nodes. IPAM is currently not supported. There is no scheduling involved, desired bridges must be precreated on all nodes. Also, ovs-cni does not configure bridges, it's up to user to connect them to L2, L3 or an overlay network. Finally please note that Open vSwitch must be installed and running on the host.
```

Comment:
could you also add sample command to create the ovs bridge?

Reply:
It is already in docs/local-cluster-example.md, I will add it to detailed usage example for arbitrary cluster too. I'd prefer the from README.md to be as clean as possible.

Reply:
"local cluster" is for developers. the README should be for users.
if you want to keep it clean, maybe refer to "getting started" page?

Reply:
covered in: deployment-on-arbitrary-cluster.md